backend/app/adapter/sqldb/short_link.go

```diff
@@ -17,6 +18,34 @@ type ShortLinkSql struct {
 	db *sql.DB
 }
 
+// UpdateOGMetaTags updates OpenGraph Meta Tags for given alias in short_link table.
+func (s *ShortLinkSql) UpdateOGMetaTags(alias string, openGraphTags metatag.OpenGraph) (entity.ShortLink, error) {
+	statement := fmt.Sprintf(`
+UPDATE "%s"
+SET "%s"=$1, "%s"=$2, "%s"=$3
+WHERE "%s"=$4;`,
+		table.ShortLink.TableName,
+		table.ShortLink.ColumnOpenGraphTitle,
+		table.ShortLink.ColumnOpenGraphDescription,
+		table.ShortLink.ColumnOpenGraphImageURL,
+		table.ShortLink.ColumnAlias,
+	)
+
+	_, err := s.db.Exec(
+		statement,
+		openGraphTags.Title,
+		openGraphTags.Description,
+		openGraphTags.ImageURL,
+		alias,
+	)
+
+	if err != nil {
+		return entity.ShortLink{}, err
+	}
+
+	return s.GetShortLinkByAlias(alias)
```

Comment:
Out of curiosity, why does this function return the short link?

Reply:
We must return something as a result of graphQL query. I thought it is better to return the ShorkLink here to verify if the update was success.
In other places we are returning the updated resource on updation:
https://github.com/short-d/short/blob/267b64bdaa039089ff49b6e63a08813daad0e917/backend/app/adapter/sqldb/short_link.go#L77